microsoft-edge/toc.yml

```
@@ -351,6 +351,12 @@
               href: extensions-chromium/publish/manage-settings.md
             - name: Contact extensions support
               href: extensions-chromium/publish/contact-extensions-team.md
+            - name: Microsoft Edge Add-ons Publish API
+              items:
+                - name: Using the Add-ons Publish API
+                  href: extensions-chromium/publish/api/using-addons-publish-api.md
+                - name: Add-ons Publish API Reference
```

```suggestion
                - name: Publish Add-ons API Reference
```
ok for a short version of the article title?  In the context of the TOC, it could be as short as just "API Reference".  It's next to the conceptual article's short title, and the parent node's title.